app/src/lib/stats/stats-database.ts

```diff
@@ -484,6 +484,42 @@ export interface IDailyMeasures {
    * failed" dialog.
    */
   readonly checksFailedDialogRerunChecksCount: number
+
+  /** The number of "approved PR" notifications the user received */
+  readonly pullRequestReviewApprovedNotificationCount: number
+
+  /** The number of "approved PR" notifications the user clicked */
+  readonly pullRequestReviewApprovedNotificationClicked: number
+
+  /**
+   * The number of times the user decided to switch to the affected pull request
+   * from the "approved PR" dialog.
+   */
+  readonly pullRequestReviewApprovedDialogSwitchToPullRequestCount: number
+
+  /** The number of "commented PR" notifications the user received */
+  readonly pullRequestReviewCommentedNotificationCount: number
+
+  /** The number of "commented PR" notifications the user clicked */
+  readonly pullRequestReviewCommentedNotificationClicked: number
+
+  /**
+   * The number of times the user decided to switch to the affected pull request
+   * from the "commented PR" dialog.
+   */
+  readonly pullRequestReviewCommentedDialogSwitchToPullRequestCount: number
+
+  /** The number of "changes requested" notifications the user received */
+  readonly pullRequestReviewChangesRequestedNotificationCount: number
+
+  /** The number of "changes requested" notifications the user clicked */
+  readonly pullRequestReviewChangesRequestedNotificationClicked: number
+
+  /**
+   * The number of times the user decided to switch to the affected pull request
+   * from the "changes requested" dialog.
+   */
+  readonly pullRequestReviewChangesRequestedDialogSwitchToPullRequestCount: number
```

Comment:
I wondered if I should've used the type `PullRequestReviewStatField` to build the specific interface for those metrics, something like…
```typescript
type IDailyMeasures = {
...
} & {
  [K in PullRequestReviewStatField]: number
}
```

It feels a bit hacky, but OTOH it's just 3 lines 😂

Reply:
Only pause it gives me is that it is currently very readable. (and following convention.. but if there is a better convention :D)

Reply:
Only pause it gives me is that it is currently very readable. (and following convention.. but if there is a better convention :D)